subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -508,6 +517,25 @@ void ull_adv_sync_offset_get(struct ll_adv_set *adv)
 	LL_ASSERT(!ret);
 }
 
+#if IS_ENABLED(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
+void ull_adv_sync_update(struct ll_adv_sync_set *sync, bool evt_extend,
+			 uint32_t update_us)
```

Comment:
`evt_extend` informs if duration of an event should be extended of shortened. The other option is to use `int32_t` and if `update_us < 0` use it to shorten event duration.

Reply:
Ah, this is my bad. I missed the `!` on the second line. 

I'm not too familiar with the ticker function, but I see that it uses both a plus and a negative argument (can both be set? Doesn't seem be any check in the implementation of the ticker). Perhaps this function should have both as well (i.e. a plus and a minus argument) to better match the ticker_update function (as this is basically just a wrapper around that). 

If both cannot (or should never) be set at the same time (i.e. a plus and a minus), then I like the idea of the signed parameter instead, giving that it can still handle the needed value range.

Reply:
I'll go with two `uint32_t` arguments `slot_plus_us` and `slot_minus_us` because:
- this is more optimal way, less instructions are generated due to lack of conditional statements (I should have gone for that in first place),
- I want to avoid types conversion because it will introduce unnecessary instructions

Reply:
Changed according the above description